cmd/agent/main.go

```diff
@@ -63,6 +63,7 @@ type config struct {
 	QueueServingPort       int    `split_words:"true"`
 	UserPort               int    `split_words:"true"`
 	RevisionTimeoutSeconds int    `split_words:"true"`
+	ComponentPort          int    `split_words:"true"`
```

Comment:
any reason change to use the env instead of argument on line 36?

Reply:
Its not getting passed as argument. So used environment. We will discuss it in chat.